<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>751</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter un accuser</string>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prenom">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Nom">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter_button">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>110</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Quitter_button">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLabel" name="affaire_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Affaire juridique</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_affaire">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="etat_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>290</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>libre</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>290</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Prisonier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic">
       <property name="geometry">
        <rect>
         <x>-120</x>
         <y>140</y>
         <width>131</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget_7">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>330</y>
         <width>391</width>
         <height>171</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_19">
        <attribute name="title">
         <string>Prisonier</string>
        </attribute>
        <widget class="QLabel" name="etat_label_27">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>institute prisonnier:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_affaire_20">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="etat_label_28">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>date de suspension</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_11">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="etat_label_29">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>date de suspension</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLabel" name="etat_label_30">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin d'avocat</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_affaire_21">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_20">
        <attribute name="title">
         <string>Libre</string>
        </attribute>
        <widget class="QLabel" name="label_30">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de presence</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_12">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>30</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_31">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_affaire_22">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_32">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>rechercher</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_15">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>100</y>
           <width>97</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Oui</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_16">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>100</y>
           <width>97</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Non</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Affichier</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_aff">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Affichier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>621</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>671</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier un accuser</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ID_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prenom_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Nom_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>200</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLabel" name="affaire_label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Affaire juridique</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_affaire_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="etat_label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>280</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>libre</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>280</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Prisonier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>update</string>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>310</y>
         <width>391</width>
         <height>171</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_17">
        <attribute name="title">
         <string>Prisonier</string>
        </attribute>
        <widget class="QLabel" name="etat_label_23">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>institute prisonnier:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_affaire_17">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="etat_label_24">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>date de suspension</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_9">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="etat_label_25">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>date de suspension</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLabel" name="etat_label_26">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>121</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin d'avocat</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_affaire_18">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_18">
        <attribute name="title">
         <string>Libre</string>
        </attribute>
        <widget class="QLabel" name="label_27">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de presence</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_10">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>30</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_affaire_19">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_33">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>rechercher</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_17">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>100</y>
           <width>97</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Oui</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_18">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>100</y>
           <width>97</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Non</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Chercher</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>821</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher accusee</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Text parole</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_ID_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>751</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'accusee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>540</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter Pdf</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>SMS</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_ID_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>781</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
